src/modules/fancyzones/editor/FancyZonesEditor/CanvasEditor.xaml.cs

```diff
@@ -13,6 +13,9 @@ namespace FancyZonesEditor
     /// </summary>
     public partial class CanvasEditor : UserControl
     {
+        // Non-localizable strings
+        private const string PropertyUpdateLayout = "UpdateLayout";
```

Comment:
All these strings are IDs, so let make that clear
`PropertyUpdateLayout` -> `PropertyUpdateLayoutID`